### ISSUE TITLE
Syntax Warning for invalid escape sequences

### ISSUE THREAD

**Issue by rene-oromtz** [canonical/checkbox#2532]
### Bug Description

`benchmarks/system/cpu_on_idle` and `benchmarks/system/disk_on_idle` are reporting a syntax error in modern Python version. 
To prevent future issues, we should consider using raw strings or double escaping the sequence.  

### Cert-blocker Test Case

- [ ] cert-blocker

### To Reproduce

1. Run any test plan that contains `benchmarks/system/cpu_on_idle` or `benchmarks/system/disk_on_idle`
2. Another option is to run directly the command for those tests. 
3. Syntax error will show

### Expected Result

No warnings on SyntaxErrors

### Actual Result

Syntax Error received on both test cases

### Environment

- OS: Resolute Racoon
- Kernel: 7.0.0-15-generic

### Relevant log output

```shell
<string>:1: SyntaxWarning: "\S" is an invalid escape sequence. Such sequences will not work in the future. Did you mean "\\S"? A raw string is also an option.
99.959 %`


<string>:1: SyntaxWarning: "\S" is an invalid escape sequence. Such sequences will not work in the future. Did you mean "\\S"? A raw string is also an option.
0.0 %
```

**Comment by rene-oromtz** [comment]
Closed as duplicate of #2477
